frontend/src/components/Repository.vue

```diff
@@ -128,7 +128,10 @@
                 <template v-for="file in selectedModel.files">
                   <a :key="file.path" class="button" :href="`${filesURL}${file.path}`">{{ file.format }}</a>&nbsp;
                 </template>
-                <br><br>
+                <div class="notification">
```

Comment:
There is a [Bulma spacing helper class](https://bulma.io/documentation/helpers/spacing-helpers/) that could be used instead of the custom CSS:

```suggestion
                <div class="notification mt-4">
```